testSrc/unit/io/flutter/run/daemon/DaemonApiTest.java

```diff
@@ -74,7 +75,7 @@ public void canEnableDeviceEvents() throws Exception {
     checkLog("{\"method\":\"device.enable\",\"id\":0}");
```

Comment:
an you add a test case matching the new edge cases that are now handled correctly?

Reply:
👍 will do